roles/serviceassurance/tasks/component_elasticsearch.yml

```diff
@@ -1,30 +1,39 @@
 - name: Set default ElasticSearch manifest
   set_fact:
     elasticsearch_manifest: |
-      apiVersion: logging.openshift.io/v1
+      apiVersion: elasticsearch.k8s.elastic.co/v1beta1
       kind: Elasticsearch
       metadata:
         name: elasticsearch
         namespace: '{{ meta.namespace }}'
       spec:
-        managementState: Managed
-        nodeSpec:
-          image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.2.8
-          resources:
-            limits:
-              memory: 4Gi
-            requests:
-              memory: 1Gi
-        redundancyPolicy: {{ elasticsearch_redundancy_policy }}
-        nodes:
-          - nodeCount: {{ elasticsearch_node_count }}
-            roles:
-              - client
-              - data
-              - master
-            storage:
-              storageClassName: ""
-              size: 10Gi
+        http:
+          tls:
+            certificate:
+              secretName: 'elasticsearch-es-cert'
+        nodeSets:
+          - config:
+              node.data: true
+              node.ingest: true
+              node.master: true
+              node.store.allow_mmap: false
+            count: {{ elasticsearch_node_count }}
+            name: default
+            podTemplate:
+              spec:
+                containers:
+                  - name: elasticsearch
+                    resources:
+                      limits:
+                        cpu: '2'
+                        memory: 4Gi
+                      requests:
+                        cpu: '1'
+                        memory: 4Gi
+                volumes:
```

Comment:
This will need to get swapped to a PVC based storage setup since that's what our other setups assume will happen. Still here since it was working for testing in upshift.

Reply:
I'm going to work on this now since it's basically the last change required before I pull this out of Draft.

Reply:
Complete